DPG/Tasks/AOTTrack/qaMatchEff.cxx

```diff
@@ -3363,6 +3364,13 @@ struct qaMatchEff {
         return;
       }
     }
+    if (isRejectNearByEvent) {
+      if (!collision.selection_bit(o2::aod::evsel::kNoCollInTimeRangeStandard)) {
```

Comment:
@ChunzhengLab are you sure that you have to put `!` in front? From slide 22 here https://indico.cern.ch/event/1425523/contributions/5995739/attachments/2893781/5081523/performance_vs_occupancy_AW_July_10_2024_IA_updated.pdf it seems to me that this is not the case. Can you confirm it with Igor?

Reply:
Hi Mattia, I have confirmed with Igor, this is correct. Put a `!` to reject events with the nearby event.